tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -655,52 +650,6 @@ def test_attention_mask(
     assert np.all(mask_zeros == 0)
 
 
-def test_log_deprecation_warning_with_old_config(caplog: LogCaptureFixture):
-    message = Message.build("hi there")
-
-    transformers_nlp = HFTransformersNLP(
-        {"model_name": "bert", "model_weights": "bert-base-uncased"}
-    )
-    transformers_nlp.process(message)
-
-    caplog.set_level(logging.DEBUG)
-    lm_tokenizer = LanguageModelTokenizer()
-    lm_tokenizer.process(message)
-    lm_featurizer = LanguageModelFeaturizer(skip_model_load=True)
-    caplog.clear()
-    with caplog.at_level(logging.DEBUG):
-        lm_featurizer.process(message)
-
-    assert "deprecated component HFTransformersNLP" in caplog.text
```

Comment:
We haven't removed `HFTransformersNLP` as part of this PR, so this test should stay?

Reply:
Removed `HFTransformersNLP` module so it should be ok to remove this now.